substratevm/src/com.oracle.svm.core.genscavenge/src/com/oracle/svm/core/genscavenge/YoungGeneration.java

```diff
@@ -56,8 +56,8 @@ public final class YoungGeneration extends Generation {
         this.survivorsToSpacesAccounting = new ChunksAccounting();
         for (int i = 0; i < maxSurvivorSpaces; i++) {
             int age = i + 1;
-            this.survivorFromSpaces[i] = new Space("Survivor-" + age + " From", true, age);
-            this.survivorToSpaces[i] = new Space("Survivor-" + age + " To", false, age, survivorsToSpacesAccounting);
+            this.survivorFromSpaces[i] = new Space("survivorFromSpace-" + age, true, age);
+            this.survivorToSpaces[i] = new Space("survivorToSpace-" + age, false, age, survivorsToSpacesAccounting);
```

Comment:
Why do you want to change this?

Reply:
For consistency reasons, `edenSpace` (no change), `tenuredSpace` (was `oldFromSpace`/`oldToSpace` before).

```
Heap:
  Young generation: 
    Eden: 
      edenSpace:
        aligned: 0 chunk(s), 0 byte(s)
        unaligned: 0 chunk(s), 0 byte(s)
    Survivors: 
      survivorFromSpace-1:
        aligned: 1 chunk(s), 1048576 byte(s)
        unaligned: 0 chunk(s), 0 byte(s)
        aligned chunks:
          0x00007fa72d400000 (0x00007fa72d401040-0x00007fa72d4e54f0)
      survivorToSpace-1:
        aligned: 0 chunk(s), 0 byte(s)
        unaligned: 0 chunk(s), 0 byte(s)
      [...]
      survivorFromSpace-15:
        aligned: 0 chunk(s), 0 byte(s)
        unaligned: 0 chunk(s), 0 byte(s)
      survivorToSpace-15:
        aligned: 0 chunk(s), 0 byte(s)
        unaligned: 0 chunk(s), 0 byte(s)
  Old generation: 
    tenuredSpace:
      aligned: 1 chunk(s), 1048576 byte(s)
      unaligned: 1 chunk(s), 131088 byte(s)
      aligned chunks:
        0x00007fa737d00000 (0x00007fa737d01040-0x00007fa737d013b8)
      unaligned chunks:
        0x00007fa73872e000 (0x00007fa73872e038-0x00007fa73874e048)
```